frontend/.agents/skills/email-best-practices/references/deliverability.md

```
@@ -0,0 +1,121 @@
+
+Maximizing the chances that your emails are delivered successfully to the recipients.
+
+## Email Authentication
+
+**Required by Gmail/Yahoo/Microsoft** - unauthenticated emails will be rejected or spam-filtered.
+
+### SPF (Sender Policy Framework)
+
+Specifies which servers can send email for your domain.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)